inpars/cpo_finetune.py

```diff
@@ -39,6 +39,13 @@ def __post_init__(self):
         self.tokenizer = AutoTokenizer.from_pretrained(self.model_name_or_path)
         if self.tokenizer.pad_token_id is None:
             self.tokenizer.add_special_tokens({'pad_token': '[PAD]'})
```

Comment:
we no longer need this line do we

Reply:
Why do you think so?

Reply:
In the following line we set the pad_token property directly.
i also tried running without this line and had no issues